lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/domain/DrivingPermitForm.java

```diff
@@ -34,26 +37,31 @@ public DrivingPermitForm() {}
 
     @JsonCreator
     public DrivingPermitForm(
-            @JsonProperty(value = "passportNumber", required = true) String passportNumber,
+            @JsonProperty(value = "drivingLicenceNumber", required = true)
+                    String drivingLicenceNumber,
             @JsonProperty(value = "surname", required = true) String surname,
             @JsonProperty(value = "forenames", required = true) List<String> forenames,
             @JsonProperty(value = "dateOfBirth", required = true) LocalDate dateOfBirth,
             @JsonProperty(value = "expiryDate", required = true) LocalDate expiryDate,
-            @JsonProperty(value = "addresses", required = true) List<Address> addresses) {
-        this.passportNumber = passportNumber;
+            @JsonProperty(value = "postcode", required = true) String postcode) {
+        this.drivingLicenceNumber = drivingLicenceNumber;
         this.surname = surname;
         this.forenames = forenames;
         this.dateOfBirth = dateOfBirth;
         this.expiryDate = expiryDate;
+        List<Address> addresses = new ArrayList<>();
+        Address address = new Address();
+        address.setPostalCode(postcode);
+        addresses.add(address);
         this.addresses = addresses;
     }
 
-    public String getPassportNumber() {
-        return passportNumber;
+    public String getDrivingLicenceNumber() {
```

Comment:
Maybe worth making is permit to be in line with everywhere else in the back end

Reply:
This particular one needs to be aligned with the frontend so that it maps